rfcs/text/0040-volume-device.md

```diff
@@ -0,0 +1,138 @@
+# 0000: volume device
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+This RFC propose adding the volume device fieldset to describe volume storage devices that are removable disks such as USB, mountable virtual disks such as ISO.
```

Comment:
I'm uncertain about using `volume.*` for the field set name if the intended use is specifically removable disks. A storage volume can be removable disk, but there are other types of physical and logical volumes users may want to describe.

Reply:
Hi @ebeahan I changed description to also include hard disks, for our requirement could also include non-removable disks.
https://github.com/elastic/ecs/pull/2201/commits/fe9c40c303d679cf9b29718df58c8284042f9a58